homeassistant/components/modbus/modbus.py

```diff
@@ -49,8 +51,9 @@ def modbus_setup(
     hass, config, service_write_register_schema, service_write_coil_schema
 ):
     """Set up Modbus component."""
-    hass.data[DOMAIN] = hub_collect = {}
 
+    # hass.data.setdefault(DOMAIN, {})
```

Comment:
I think we should either enable this code or remove it before merging into core.

Reply:
good catch, removed.